packages/lambdas/src/document-uploader.ts

```diff
@@ -0,0 +1,39 @@
+import { documentUploaderHandler } from "@metriport/core/external/aws/lambda-logic/document-uploader";
+import { S3Event } from "aws-lambda";
+import { capture } from "./shared/capture";
+import { getEnvOrFail } from "./shared/env";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const apiUrl = getEnvOrFail("API_URL");
+const destinationBucket = getEnvOrFail("MEDICAL_DOCUMENTS_DESTINATION_BUCKET");
+const region = getEnvOrFail("AWS_REGION");
+
+export const handler = async (event: S3Event) => {
+  if (event.Records[0]) {
```

Comment:
Couldn't the lambda run with multiple records, like for more then one file that finished uploading at the same time?

Also, nit: when we have to refer to the same thing multiple times its a good idea to create a const for it (e.g., `event.Records[0]`)

Reply:
I was thinking that since this lambda only gets triggered on file uploads, which have to be done individually thru the signed URLs, then it's always going to be `event.Records.length = 1`, but definitely safer to just iterate thru all possible records + apparently, it is indeed possible that Lambda puts these events in a batch if multiple files get uploaded to the bucket at around the same time. So, good catch 🥇